txpipe/__init__.py

```diff
@@ -30,3 +30,4 @@
 from .masks import TXSimpleMask
 from .metadata import TXTracerMetadata
 from .convergence import TXConvergenceMaps
+from .map_tests import TXMapCorrelations
```

Comment:
wonder if the map_tests name can be a little more descriptive, I think map_correlation could be better?

Reply:
changed.